examples/210-embedding_models.ipynb

```diff
@@ -0,0 +1,1128 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    pipeline = Pipeline(model=model, transforms=[lag_transform], horizon=HORIZON)
</pre><p>Have you tried adding <code>SegentEncoderTransform</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
We need to improve pipeline with embedding features, not baseline

I have not tried